cmd/gardener-extension-provider-local/app/app.go

```diff
@@ -152,6 +161,7 @@ func NewControllerManagerCommand(ctx context.Context) *cobra.Command {
 			controllercmd.PrefixOption("infrastructure-", infraCtrlOpts),
 			controllercmd.PrefixOption("worker-", &workerCtrlOptsUnprefixed),
 			controllercmd.PrefixOption("service-", serviceCtrlOpts),
+			controllercmd.PrefixOption("backupbucket-", localBackupBucketOptions),
```

Comment:
Can you extend the Helm chart for these new flags (https://github.com/gardener/gardener/tree/master/charts/gardener/provider-local/extension), please?